Portfolio.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE = html>
+<html lang="en">
+<head>
+  	<meta charset="utf-8">
+  	<title>Jenny Allton-Twist</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="Portfoliostyle.css" type="text/css" rel="stylesheet">	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+	<nav>
+		<ul>
+			<li><a href="#home">Home</a></li>
+			<li><a href="#portfolio">Portfolio</a></li>
+			<li><a href="#aboutme">About Me</a></li>
+			<li><a href="#contact">Contact</a></li>
+		 </ul>
+		 <div>
+			 
+	 <!-- <div><img src="" alt=""/> Place for logo -->
+		 </div>
+	</nav>
+
+
+
+	<div class="intropage" id="home">
+		<div class="container">
+			<h1>JENNY ALLTON-TWI<span clasmate s="spinner">S</span>T</h1>
```

Comment:
clasmate? this is not valid html. I assume you meant `class="spinner"`

Reply:
typed in the editor instead of in the terminal by accident?

Reply:
Typo as Rayna pointed out - I must have been trying to open textmate and wondering why it wasn't typing inthe terminal... doh